src/.env.sample

```diff
@@ -51,3 +51,9 @@ SUPPORTED_HTTP_TYPES="GET POST PUT PATCH DELETE"
 
 # Required base URL for the user service
 USER_SERVICE_BASE_URL=http://localhost:7001
+
```

Comment:
@adithyadinesh0412 Before we review this PR again, please raise and get PRs merged to the main repos with their route configs